libsolidity/interface/InterfaceHandler.cpp

```diff
@@ -51,7 +51,8 @@ string InterfaceHandler::abiInterface(ContractDefinition const& _contractDef)
 		Json::Value method;
 		method["type"] = "function";
 		method["name"] = it.second->declaration().name();
-		method["constant"] = it.second->isConstant();
```

Comment:
Will leave this in as a deprecated field which we can remove in the breaking release. (So it will have both `constant` and `view` set)